src/main/java/org/rumbledb/expressions/flowr/Clause.java

```diff
@@ -27,13 +27,13 @@
  * GRAMMAR:flowrExpression
  */
```

Comment:
I believe this comment may be outdated and misleading. I'd suggest removing it.

Reply:
Good catch. DONE.